yumljira/assets/store/index.js

```diff
@@ -0,0 +1,12 @@
+import Vue from 'vue'
```

Comment:
I think this is useless

Reply:
No. We need that import, because it's store state and without that build just break.

Reply:
Am yes, but *Do we need this file*

Reply:
Oh no. Sorry I don't look on path :D

Reply:
BTW It's my bad I should be more specific.